charts/kof-mothership/templates/victoria/vmcluster.yaml

```diff
@@ -29,7 +29,7 @@ spec:
         spec:
           resources:
             requests:
-              storage: 2Gi
+              storage: {{ .Values.victoriametrics.vmcluster.vmselect.storage.size }}
```

Comment:
Could you put a comment with the documentation link to understand what is this effecting, and what is the difference with the other storage size for vmstorage, please?